src/components/KCard/KCard.vue

```diff
@@ -1,244 +1,122 @@
 <template>
-  <section
-    :aria-describedby="contentId || undefined"
-    :aria-label="title ? title : undefined"
-    :aria-labelledby="!title && slots.title ? titleId : undefined"
-    class="kong-card"
-    :class="[borderVariant, {'hover': hasHover, 'kcard-shadow': hasShadow }]"
-  >
+  <section class="k-card">
```

Comment:
Let's change this to a `div`; a host app should have control over its HTML hierarchy

Reply:
Done